docs/user-guide-state-configure-interface-mtu.md

```diff
@@ -0,0 +1,69 @@
+# Create a Dummy Interfaces and Set its MTU
```

Comment:
You should mention this is a 'tutorial'/'step by step' ... It is not clear what it is until you start reading it.

Reply:
Done in all user guides